docs/schema.md

```diff
@@ -117,19 +117,27 @@ If the parent type is part of the same namespace as the child type, then you can
 
 Specifies the data type and attributes that are needed to define entities of this type. 
 
-Each attribute defined in a `shape` must include a `type` specification, with a value that names one of the [Cedar supported data types](syntax-datatypes.md). The following example shows a simple shape.
+The top level `shape` of an entity must have type `Record`. The follwoing example shows a simple Entity specification with a shape:
 
```

Comment:
Change to "The following [notice typo fix] example shows a simple specification of the `User` entity type:"